gn3/db/traits.py

```diff
@@ -86,146 +86,296 @@ def retrieve_trait_dataset_name(
     implemented at
     https://github.com/genenetwork/genenetwork1/blob/master/web/webqtl/base/webqtlDataset.py#L140-L169
     """
+    table_map = {
```

Comment:
**nitpick:** for constants, I normally uppercase them. See [this](https://www.python.org/dev/peps/pep-0008/#id36). Better yet, when I can, I'd underscore them. Not necessarily useful convention, but IMHO makes things easier to read...

Reply:
> **nitpick:** for constants, I normally uppercase them. See [this](https://www.python.org/dev/peps/pep-0008/#id36). Better yet, when I can, I'd underscore them. Not necessarily useful convention, but IMHO makes things easier to read...

I noticed.

My thoughts, however, at least as regards this particular case, the `table_map` variable, in as much as it does not change, is a local variable, and thus, the scope of concern is within just the function it is defined.

I also ran into the issue where if one uses uppercase variables, the linter is not happy with them, so I am choosing to avoid uppercase variable names unless absolutely necessary.